lib/tldr.js

```diff
@@ -130,6 +130,10 @@ class Tldr {
         // If found in first try, render it
         if (!content) {
           // If not found, try to update
+          if (this.config.skipUpdateWhenPageNotFound === true) {
+            console.log('Skipping update of cache because option skipUpdateWhenPageNotFound is set.');
```

Comment:
Does that get printed every time a page is not found? That's not as annoying as an update, but still. If the user manually set's this config variable, I don't think we need a reminder every time.